### DISCUSSION TITLE
mike + mkdocs-gen-files

### DISCUSSION THREAD

**Discussion by dbbvitor** [squidfunk/mkdocs-material#4733] [Q&A] [answered]
Hi! I'm having some trouble trying to set up mike with the mkdocs-gen-files pluging. Has anyone managed to have both working together?

If I deploy or serve with `mike` I get versioned pages but no auto generated pages:
```bash
mike serve
```
If I deploy or serve with `mkdocs` I get auto generated pages but with no versioning:
```bash
mkdocs serve
```

Here is the mkdocs.yaml in question: [my repo](https://github.com/dbbvitor/gramophone-spotify-pipeline/blob/feat/auto-docs-generator/mkdocs.yml)
Here is the documentation I followed to set up mkdocs-gen-files: [mkdocstring docs](https://mkdocstrings.github.io/recipes/#generate-pages-on-the-fly)
Here is the documentation I followed to set up mike: [material for mkdocs docs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/?h=mike)

**Comment by squidfunk** [comment]
This is essentially a `mike` or `mkdocs-gen-files` question, as it doesn't seem to be exclusively related to Material for MkDocs. I recommend asking the maintainers. If you make any progress, it would be great if you could share it with other users.

**Comment by dbbvitor** [comment, marked as answer]
Hum... turns out `mike serve` doesn't show the local version of the docs as `mkdocs serve` does.... It shows the version currently on the gh-pages branch
